assets/app/features/upgrade/upgrade.constants.js

```diff
@@ -3,8 +3,7 @@
     angular
         .module('crowbarApp.upgrade')
         .constant('ADDONS_PRECHECK_MAP', {
-            'ha': ['clusters_healthy'],
-            'ceph': ['ceph_healthy']
```

Comment:
Note there there's this new group called `ceph_status` (which could contain crowbar_ceph_present error)

Reply:
yup, updating the PR right now.

Reply:
should be fixed now. see my assumptions comment below.